recipes/grpc-cpp/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Is this file in the tarball? Then you can remove it

Reply:
Ah, I think I have misread the packaging instructions. Will do.